### PR TITLE
refactor article for AzureAD authentication

### PR DESCRIPTION
**Pull request by ighristov**

### CODE REVIEW

File: implementer-guide/setup/how-to-use-azure-ad-to-login-to-trs.md [hunk 1]
```diff
@@ -1,104 +1,138 @@
...
+- Switch to Report Server's Configuration page and paste it into the **Metadata URI** textbox. Append the following chunk to the URI: ````?appid={your-application-id}```` where ````{your-application-id}```` is the **Application (client) ID** of the Azure application. You can see it in the Essentials panel in the Overview tab. Finally, the contents of the **Metadata URI** textbox should look like this: 
+````https://login.microsoftonline.com/{your-directory(tenant)-id}/federationmetadata/2007-06/federationmetadata.xml?appid={your-application-id}````
+- Enter the URL of the Telerik Report Server application in the **Relying Party ID** textbox. Its value must be the same as the value of the Redirect URI configured in step 5.
 
-![AzureAD - application setup](../../images/report-server-images/HowToLoginUsingADFS/azure_ad_step6.png)
+With this step the authentication configurations for both the Azure application and Telerik Report Server are complete. At this point, you should be able to log in to Report Server using the web interface, by clicking on the blue button "**Active Directory Credentials**" on the login screen.
```
Comment (elkinmilen) [inline implementer-guide/setup/how-to-use-azure-ad-to-login-to-trs.md:92]: "At this point..." - this leaves the impression, that we are good to log in, but we still have no users. Maybe rephrase the sentence that we are good to go once we add the users to RS.

**Review by elkinmilen — APPROVED** [review]
Made some minor grammar fixes based on a tool feedback & added a single comment. Thank you!

File: implementer-guide/setup/how-to-use-azure-ad-to-login-to-trs.md (hunk 1 above)
Comment (ighristov) [reply implementer-guide/setup/how-to-use-azure-ad-to-login-to-trs.md:92]: Correct, I assumed the users are already added to RS, which might not be true. I changed it to:
The configuration of the Azure application and Telerik Report Server is now complete. In order to be successfully logged-in in Report Server Manager web app, ensure that the Azure AD users are added as Telerik Report Server users, as described in section **Report Server Users** below.
